### PR TITLE
DOCSUP-36377-training-docs-update

### PR DESCRIPTION
**Pull request by lehasm**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/client/__init__.py [hunk 1]
```diff
@@ -1634,16 +1634,17 @@ def update_pool(self, pool_id: str, pool: Pool) -> Pool:
 
     @add_headers('client')
     def archive_training(self, training_id: str) -> Training:
-        """Sends training to archive
+        """Archives a training pool.
```
Comment (alexdrydew) [inline src/client/__init__.py:1637]: I would rather not use collocation "training pool", as there might be confusion between `Training` and legacy pool of training type

File: src/client/__init__.py [hunk 2]
```diff
@@ -1658,17 +1659,17 @@ def archive_training(self, training_id: str) -> Training:
 
     @add_headers('client')
     def archive_training_async(self, training_id: str) -> Optional[operations.TrainingArchiveOperation]:
-        """Sends training to archive, asynchronous version
+        """Archives a training pool. Sends an asynchronous request to Toloka.
```
Comment (alexdrydew) [inline src/client/__init__.py:1662]: "training pool" collocation, see previous comment

File: src/client/__init__.py (hunk 1 above)
Comment (alexdrydew) [reply src/client/__init__.py:1637]: P.S. if you agree with me on this please replace "training pool" collocation in the other changes as well

File: src/client/training.py [hunk 3]
```diff
@@ -11,54 +11,66 @@
...
 class Training(BaseTolokaObject):
-    """Training pool
+    """A training pool.
 
-    Allows:
-     - Select for the main pool only those Tolokers who successfully complete the training tasks.
-     - Practice Tolokers before the main pool and figure out how to respond correctly.
+    A training pool contains tasks with known solutions and hints for Tolokers. Use training pools:
+    - To train Tolokers so they solve general tasks better.
+    - To select for general pool Tolokers who successfully completed training tasks.
```
Comment (alexdrydew) [inline src/client/training.py:18]: Sounds just a bit confusing to me, maybe something like
`To select Tolokers who successfully completed training tasks for general pool labeling.`?

File: src/client/training.py [hunk 4]
```diff
@@ -11,54 +11,66 @@
...
+
+    To link a trining pool to a general pool set the
```
Comment (alexdrydew) [inline src/client/training.py:20]: Typo:
```suggestion
    To link a training pool to a general pool set the
```

File: src/client/training.py [hunk 5]
```diff
@@ -11,54 +11,66 @@
...
+        project_id: The ID of the project containing the training.
+        private_name: The training pool name. It is visible to the requester only.
+        may_contain_adult_content: The presence of adult content.
+        assignment_max_duration_seconds: Time limit to complete a task suite.
+            Take into account loading a page with a task suite and sending responses to the server. It is recommended that you set at least 60 seconds.
+        mix_tasks_in_creation_order:
+            * True — Tasks are grouped in suites in the order they were created.
+            * False — Tasks are chosen for a task suite in a random order.
+
+            Default: False.
```
Comment (alexdrydew) [inline src/client/training.py:36]: As you suggested earlier the default value is `True` (the default is on the backend side, client-side the default is `None`, which means the field is not added to the request body)

File: src/client/training.py [hunk 6]
```diff
@@ -11,54 +11,66 @@
...
+        shuffle_tasks_in_task_suite:
+            * True — Tasks from a task suite are shuffled on the page.
+            * False — Tasks from a task suite are placed on the page in the order they were created.
+
+            Default: False.
```
Comment (alexdrydew) [inline src/client/training.py:41]: As you suggested earlier the default value is `True` (the default is on the backend side, client-side the default is `None`, which means the field is not added to the request body)

File: src/client/training.py [hunk 7]
```diff
@@ -11,54 +11,66 @@
...
+        training_tasks_in_task_suite_count: The number of training tasks in one suite.
```
Comment (alexdrydew) [inline src/client/training.py:42]: Just to be more specific
```suggestion
        training_tasks_in_task_suite_count: The number of training tasks in one task suite.
```

File: src/client/training.py [hunk 8]
```diff
@@ -11,54 +11,66 @@
...
+            * False — Instruction, specified in the `public_instructions` parameter, are used.
+
+            Default: False.
+        public_instructions: Instructions for Tolokers used when the `inherited_instructions` parameter is False. Describe in the instructions how to complete training tasks.
+            You can use HTML markup inside `public_instructions`.
+        metadata: A dictionary with metadata.
+        owner: A training pool owner.
+        id: The ID of the training pool. Read only.
+        status: A training pool status. Read only.
+        last_close_reason: A reason why the training pool was closed last time.
```
Comment (alexdrydew) [inline src/client/training.py:57]: This field is also read-only

File: src/client/__init__.py (hunk 1 above)
Comment (lehasm) [reply src/client/__init__.py:1637]: OK. Changed to just "training".

File: src/client/training.py (hunk 3 above)
Comment (lehasm) [reply src/client/training.py:18]: Eventually changed to: 
To select Tolokers who successfully completed training tasks and to give them access to a general pool.

File: src/client/training.py (hunk 4 above)
Comment (lehasm) [reply src/client/training.py:20]: fixed

File: src/client/training.py [hunk 9]
```diff
@@ -11,54 +11,66 @@
...
         Attributes:
-            MANUAL: Closed by the requester.
-            EXPIRED: Reached the date and time set in will_expire.
-            COMPLETED: Closed automatically because all the pool tasks were completed.
-            NOT_ENOUGH_BALANCE: Closed automatically because the Toloka account ran out of funds.
-            ASSIGNMENTS_LIMIT_EXCEEDED: Closed automatically because it exceeded the limit on assigned task suites
-                (maximum of 2 million).
-            BLOCKED: Closed automatically because the requester's account was blocked by a Toloka administrator.
+            MANUAL: A training was closed by a requester.
+            EXPIRED: The lifetime of the linked pool has expired.
```
Comment (alexdrydew) [inline src/client/training.py:69]: Reached out to backend team to clarify about EXPIRED and NOT_ENOUGH_BALANCE statuses:
New training will not have these statuses so these are either lagecy statuses or a mistake in the old version of API documentation.
Can you please update these docstrings for one more time?

Comment (lehasm) [reply src/client/training.py]: Removed EXPIRED and NOT_ENOUGH_BALANCE statuses from docstring.
(Did you mean that?)

Comment (lehasm) [reply src/client/training.py]: @alexdrydew 
Checks passed. )

**Review by alexdrydew — APPROVED** [review]

**Pull request merged**